public/util/lex.ts

```diff
@@ -33,129 +75,288 @@ export function variablesToLexLanguage(variables: Variable[]): Lex {
     icon: '<i class="fa fa-filter" />',
     suggestions: suggestions,
   }).branch(
-    Lex.from("value", TextEntryState, {
+    Lex.from("relation", DistilRelationState, {
       ...TransitionFactory.valueMetaCompare({ type: TEXT_FILTER }),
-    }),
-    Lex.from("value", TextEntryState, {
+    }).branch(Lex.from("value", TextEntryState)),
```

Comment:
I wish we could populate the categorical variables with its values.

Reply:
Would require either pulling down all of the summaries (very slow on large datasets, which is why we explicitly stopped doing that,) or dramatically changing the variables end point to relay that information (out of scope for this iteration of the work. Need UX feedback first.)

Reply:
Yes, I noticed that in my testing as well.  Is there something particular with lexbar that would block this?